source/unified-test-format/unified-test-format.md

```diff
@@ -2745,15 +2751,47 @@ is not always feasible or relevant for a test to exhaustively specify all fields
 time in `command` for [CommandStartedEvent](#expectedEvent_commandStartedEvent)).
 
 When the expected value is an array, test runners MUST differentiate between an array of values, which may be documents,
-(e.g. `distinct`) and an array of root-level documents (e.g. `find`, `aggregate`). For example, the following array of
-documents would not match if returned by `distinct`, but would match if returned via `find` (after iterating the
-cursor):
+(e.g. `distinct`) and an array of root-level documents derived from an iterable result (e.g. `find`, `aggregate`). For
+example, the following array of documents would not match if returned by `distinct`, but would match if returned via
+`find` (after iterating the cursor):
 
 ```yaml
 expected: [ { x: 1 }, { x: 2 } ]
 actual: [ { x: 1, y: 1 }, { x: 2, y: 2 } ]
 ```
 
+##### Examples of Root-level Documents
```

Comment:
@stIncMale: I went through the spec and inferred what root-level contexts I could. Since operations aren't entirely defined there, and PHP is behind on many specs (e.g. CSOT), I have no idea if it's complete; however, it at least adds a point where the full list can be maintained as other contexts are found.

I notably removed the "includes but not limited to" line from the original text and gave this its own section.